frontend/packages/db-structure/src/parser/drizzle/index.test.ts

```diff
@@ -0,0 +1,757 @@
+          description: text('description').default("user's description"),
+        });
+      `)
+
+      expect(value).toEqual(parserTestCases['default value as string'])
+    })
+
+    it('default value as integer', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial, integer } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+          age: integer('age').default(30).notNull(),
+        });
+      `)
+
+      const expected = userTable({
+        columns: {
+          age: aColumn({
+            name: 'age',
+            type: 'integer',
+            default: 30,
+            notNull: true,
+          }),
+        },
+      })
+
+      expect(value).toEqual(expected)
+    })
+
+    it('default value as boolean', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial, boolean } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+          active: boolean('active').default(true).notNull(),
+        });
+      `)
+
+      const expected = userTable({
+        columns: {
+          active: aColumn({
+            name: 'active',
+            type: 'boolean',
+            default: true,
+            notNull: true,
+          }),
+        },
+      })
+
+      expect(value).toEqual(expected)
+    })
+
+    it('column comment', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial, text } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+          description: text('description').$comment('this is description'),
+        });
+      `)
+
+      expect(value).toEqual(parserTestCases['column comment'])
+    })
+
+    it('table comment', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+        }).$comment('store our users.');
+      `)
+
+      expect(value).toEqual(parserTestCases['table comment'])
+    })
+
+    it('unique constraint', async () => {
+      const { value } = await processor(`
+        import { pgTable, serial, varchar, unique } from 'drizzle-orm/pg-core';
+
+        export const users = pgTable('users', {
+          id: serial('id').primaryKey(),
+          email: varchar('email', { length: 255 }).unique(),
+        });
+      `)
+
+      const expected = userTable({
+        columns: {
+          email: aColumn({
+            name: 'email',
+            type: 'varchar(255)',
+            notNull: true,
+          }),
+        },
+        indexes: {
+          users_email_unique: anIndex({
+            name: 'users_email_unique',
+            columns: ['email'],
+            unique: true,
+          }),
+        },
```

Comment:
Please fix `.unique()` as it seems to generate only constraints. A separate PR is fine.

https://orm.drizzle.team/docs/indexes-constraints#unique

Reply:
I understand! 
I will fix this!Thank you!